CLIP-1566: Make it possible to configure images/tags of auxiliary containers

bianchi2 opened the pull request:
Fixes https://github.com/atlassian/data-center-helm-charts/issues/411

A small change to values and templates to be able to pass custom image and tag to nfs permission fixes init container and fluentd sidecar (if enabled)

## Checklist
- [x] I have added unit tests
- [x] I have applied the change to all applicable products
- [ ] I have added the change description to the `changelog.md` and `Chart.yaml` files
- [ ] (Atlassian only) I have run the E2E test (if applicable)
- [ ] (Atlassian only) Internal Bamboo CI is passing

nanux commented:
LGTM. This is removing one of the blockers for airtight environments. Maybe adding a small paragraph to our documentation would be useful as well :)

bianchi2 commented:
@nanux I have removed one limitation and added a new section to the installation guide with instructions on how to redefine images that statefulsets containers may use.
